sdk/servicebus/azure-servicebus/stress/stress-test-resources.bicep

```diff
@@ -65,17 +61,6 @@ resource authorizationRuleNameNoManage 'Microsoft.ServiceBus/namespaces/Authoriz
 
 
 
-resource dataOwnerRoleId 'Microsoft.Authorization/roleAssignments@2018-01-01-preview' = {
-  name: guid('dataOwnerRoleId${baseName}')
```

Comment:
we dont need the role assignment step anymore?

Reply:
nope the stress infra now has role assignments pre-provisioned